packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -43,6 +43,7 @@ function BlockPattern( {
 			isEnabled={ isDraggable }
 			blocks={ blocks }
 			isPattern={ !! pattern }
+			label={ __( 'Pattern' ) }
```

Comment:
Should this always be "Pattern"?

Reply:
Good catch. I haven't thought of it because it's the pattern inserter. I'll add some changes to this.